traffic_portal/test/end_to_end/users/users-spec.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var pd = require('./pageData.js');
+var cfunc = require('../common/commonFunctions.js');
+
+describe('Traffic Portal Users Test Suite', () => {
+    const pageData = new pd();
+    const commonFunctions = new cfunc();
+    const newUser = function() {
+        return {
+            name: 'User Name',
+            username: 'username-' + commonFunctions.shuffle('abcdefghijklmonpqrstuvwxyz0123456789'),
+            email: 'user-' + commonFunctions.shuffle('abcdefghijklmonpqrstuvwxyz0123456789') + '@example.com',
+            password: commonFunctions.shuffle('abcdefghijklmonpqrstuvwxyz0123456789'),
+            role: 'admin (30)',
+            tenant: '- root'
+        };
+    };
+    const myNewUser = newUser();
+    const myOtherNewUser = newUser();
+    const repeater = 'u in ::users';
+
+    it('should register a new user', async () => {
+        console.log('Registering new user')
+        await browser.setLocation('users');
+        await pageData.registerUserButton.click();
+        await pageData.email.sendKeys(myNewUser.email);
+        await commonFunctions.selectDropdownByLabel(pageData.role, myNewUser.role); // note: this creates a new user with admin permissions
+        await commonFunctions.selectDropdownByLabel(pageData.tenant, myNewUser.tenant);
+        await pageData.sendRegistration.click();
+        // these statements are commented out because I kept getting an internal server error when running this test locally
```

Comment:
is there a bug raised for this issue?

Reply:
I'm not really sure what the issue is, could someone uncomment that line and see if the test still works for them?